config.yaml

```diff
@@ -24,7 +24,7 @@ markup:
   highlight:
     codeFences: true
     noClasses: false
-    linenos: true
+    linenos: false
```

Comment:
(I don't have strong opinions here) why do we want to remove line numbers? This also removes the ability to link to a specific lines in an output. It looks cleaner, but it makes it less user friendly.

Reply:
I looked through a few Docs references that were omitting the line numbers. If we can find someone else including these, I'm happy to include them